src/domain/operation_mode.py

```diff
@@ -119,6 +122,17 @@ def send_email(self, body, img_path):
             smtp_server.quit()
         logger.info("Email notification for %s sent!", img_path)
 
+    def actuate(self, actuator_list):
+        """Method to trigger actuators when enabled"""
+        for actuator in actuator_list:
+            if actuator.enabled:
+                if actuator.type == Actuator.ActuatorTypes.RoadSignActuator:
+                    actuator.send_command(RoadSignActuator.Commands.DisplayON)
+                elif actuator.type == Actuator.ActuatorTypes.FeederActuator:
+                    actuator.send_command(FeederActuator.Commands.Open)
+                else:
+                    raise Exception("Actuator type not known")
```

Comment:
Delegate this to an abstract method on the base class, and a concrete method implementation on the subclasses.

Reply:
> Delegate this to an abstract method on the base class, and a concrete method implementation on the subclasses.

This method needs to be correctly implemented by @stefanodellosa-personal.